### DISCUSSION TITLE
net::ERR_CONNECTION_REFUSED

### DISCUSSION THREAD

**Discussion by pydeveloper510** [socketio/socket.io#4552] [Q&A]
![image](https://user-images.githubusercontent.com/85598258/206762904-dc04c714-a0ab-460c-ac65-d2ff88eb538d.png)

This is my error on server.
When I try it locally there is no error.

what is my mistake?

Please help me.
Thank you!

**Comment by darrachequesne** [comment]
Hi! It seems the port 8080 of your server is not reachable, could you please check if there's something in between preventing the connection?

Related: https://socket.io/docs/v4/troubleshooting-connection-issues/#the-server-is-not-reachable
